test_mcp_server.py

```
@@ -0,0 +1,172 @@
+import json
+import subprocess
+import sys
+import time
+from pathlib import Path
+
+def send_mcp_message(process, message):
+    """Send an MCP message to the process."""
+    content = json.dumps(message)
+    content_bytes = content.encode('utf-8')
+    header = f"Content-Length: {len(content_bytes)}\r\n\r\n"
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)